lib/edition_filter.rb

```diff
@@ -22,6 +22,7 @@ def editions
                    .preload(revision_joins, :status, :document, :last_edited_by)
     scope = access_limited_scope(scope)
     scope = filtered_scope(scope)
+    scope = pre_release_document_type_scope(scope)
```

Comment:
I think we actually mean the inverse here: the scope actually returns the non-pre-release ones.

Reply:
Have renamed